cmd/usage.go

```
@@ -166,36 +209,42 @@ func fetchAccountsConcurrently(ctx context.Context, app *app, accounts []domain.
-		return fmt.Errorf("all accounts failed to fetch")
+	return summary, nil
+}
+
+func writeFetchSummaryPlain(errWriter io.Writer, summary fetchSummary) {
+	if errWriter == nil || !summary.hasFailures() {
+		return
 	}
 
-	if len(successes) > 0 && len(failures) > 0 {
-		fmt.Fprintf(errWriter, "\n%d/%d accounts updated successfully\n", len(successes), len(accounts))
+	fmt.Fprintln(errWriter, "\nFailed to fetch:")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)